apps/condo/domains/analytics/tasks/helpers/sendDailyStatistics.spec.js

```diff
@@ -0,0 +1,652 @@
+const { SEND_DAILY_STATISTICS_MESSAGE_TYPE } = require('@condo/domains/notification/constants/constants')
+const { Message } = require('@condo/domains/notification/utils/testSchema')
+const { makeEmployeeUserClientWithAbilities, createTestOrganizationEmployee, createTestOrganizationEmployeeRole } = require('@condo/domains/organization/utils/testSchema')
+const { makeClientWithProperty } = require('@condo/domains/property/utils/testSchema')
+const { INCIDENT_STATUS_NOT_ACTUAL, INCIDENT_CATEGORY_IDS, INCIDENT_PROBLEM_IDS } = require('@condo/domains/ticket/constants/incident')
+const { STATUS_IDS } = require('@condo/domains/ticket/constants/statusTransitions')
+const { updateTestTicket, createTestIncidentProperty } = require('@condo/domains/ticket/utils/testSchema')
+const {
+    createTestTicket, TicketStatus, createTestIncident, createTestIncidentClassifierIncident, IncidentClassifier,
+} = require('@condo/domains/ticket/utils/testSchema')
+const { makeClientWithNewRegisteredAndLoggedInUser } = require('@condo/domains/user/utils/testSchema')
+
+const { UserDailyStatistics, sendDailyMessageToUserSafely } = require('./sendDailyStatistics')
+
+
+describe('sendDailyStatistics helpers', () => {
+    let admin, statuses,
+        allowedIncidentClassifiers, notAllowedIncidentClassifiers
+
+    setFakeClientMode(index)
+    jest.setTimeout(10000)
+
+
+    beforeAll(async () => {
+        setFeatureFlag(SEND_DAILY_STATISTICS_TASK, true)
+
+        admin = await makeLoggedInAdminClient()
+        statuses = await TicketStatus.getAll(admin, {})
+
+        allowedIncidentClassifiers = await IncidentClassifier.getAll(admin, {
+            category: {
+                id: INCIDENT_CATEGORY_IDS.WATER,
+            },
+            problem: {
+                id_in: [
+                    INCIDENT_PROBLEM_IDS.NO_HOT_AND_COLD_WATER,
+                    INCIDENT_PROBLEM_IDS.NOT_HOT_WATER,
+                    INCIDENT_PROBLEM_IDS.NO_COLD_WATER,
+                ],
+            },
+        })
+        notAllowedIncidentClassifiers = await IncidentClassifier.getAll(admin, {
+            id_not_in: allowedIncidentClassifiers.map(item => item.id),
+        })
+    })
+
+    describe('class UserDailyStatistics', () => {
+        describe('should return correct statistic data for user', () => {
+            let userData = {
+                client: null,
+                organizationWithAccess1: {
+                    data: null,
+                    property: null,
+                },
+                organizationWithAccess2: {
+                    data: null,
+                    property: null,
+                },
+                organizationWithoutAccess: {
+                    data: null,
+                    property: null,
+                },
+            }
+            let justEmployeeClient
+
+            beforeEach(async () => {
+                setFeatureFlag(SEND_DAILY_STATISTICS_ORGANIZATIONS_ENABLED, false)
```

Comment:
why do you need two feature flags, seems confusing

Reply:
First feature flag allows to start a bypass by users

Second feature flag allows you to receive data for a specific organization